client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/survey/survey.scss

```diff
@@ -8,7 +8,7 @@
 		cursor: default;
 		border: 0;
 		box-shadow: none;
-		padding-right: 52px;
+		padding-right: 52px !important;
```

Comment:
Is it possible to avoid using `important`? If not, it might be better to explain why we need it here 😉

Reply:
Okay, I'll add a comment to explain that we overwrite the padding added by responsive styles from `<Card />`

Reply:
I decided to avoid the important using a media query because that's what is needed. Any other idea?